### ISSUE TITLE
`no-shadowed-elements` returning false positive for `col` within `colgroup`

### ISSUE THREAD

**Issue by MelSumner** [ember-template-lint/ember-template-lint#2740]
The `no-shadowed-elements` is returning a false positive for use of `col` within `colgroup`. 

Documentation Reference for the `col` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/col

I expect this code to exist without issue: 

```hbs
{{#each @columns as |col|}}
  {{#if col.width}}
    <col width={{col.width}} />
  {{else}}
    <col />
  {{/if}}
{{/each}}
```

**Comment by lifeart** [comment]
It seems not an issue, because here `col` param from `#each` helper is shadowing `col` html node name.

**Comment by lifeart** [comment]
closing issue as it could be resolved with a-bit different naming
